prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -125,17 +129,27 @@ else
   if [ $BEAMTYPE == "PbPb" ]; then
     [ -z ${ITS_CONFIG+x} ] && ITS_CONFIG=" --tracking-mode async"
   elif [ $BEAMTYPE == "pp" ]; then
-    ITS_CONFIG_KEY+="fastMultConfig.cutMultClusLow=1;fastMultConfig.cutMultClusHigh=2000;fastMultConfig.cutMultVtxHigh=500;ITSVertexerParam.phiCut=0.5;ITSVertexerParam.clusterContributorsCut=3;ITSVertexerParam.tanLambdaCut=0.2"
-    [ -z ${ITS_CONFIG+x} ] && ITS_CONFIG=" --tracking-mode async"
+    if [ -z $ISPILOTBEAMDATA ]; then # configuration of ITS tracking for pilot beam data
+      ITS_CONFIG_KEY+="ITSVertexerParam.phiCut=0.5;ITSVertexerParam.clusterContributorsCut=3;ITSVertexerParam.tanLambdaCut=0.2;ITSCATrackerParam.useDiamond=true;ITSCATrackerParam.diamondPos[2]=35.;ITSCATrackerParam.pvRes=10."
+      [ -z ${ITS_CONFIG+x} ] && ITS_CONFIG=" --tracking-mode sync_misaligned"
+    else
+      ITS_CONFIG_KEY+="fastMultConfig.cutMultClusLow=1;fastMultConfig.cutMultClusHigh=2000;fastMultConfig.cutMultVtxHigh=500;ITSVertexerParam.phiCut=0.5;ITSVertexerParam.clusterContributorsCut=3;ITSVertexerParam.tanLambdaCut=0.2"
```

Comment:
actually, this line should have not been there, we don't need any selection in the async mode, could you simply suppress it?

Reply:
Sure.

Reply:
Full line removed for syncmode = 0.